backend/src/controllers/squad.js

```diff
@@ -216,7 +220,6 @@ exports.deleteSquadMember = async (req, res, next) => {
       message: "User removed from the squad."
     });
   } catch (err) {
-    console.log(err);
```

Comment:
Are we logging these errors anywhere at all?

Reply:
I took out the console logs because they blow up the tests. I'm honestly not sure how to handle these. I feel like we need to figure out a better way to track responses from the API. I'm sure there's some sort of library we could add that does it for us. As in, we get a request, and every response the API sends logs the status code and the endpoint that is hit. 

These console logs are pretty useless IMO.